src/main/java/com/scalar/kelpie/modules/PostProcessor.java

```diff
@@ -37,6 +37,6 @@ protected void getSummary() {
       return;
     }
 
-    logInfo(stats.getSummary());
+    stats.getSummary();
```

Comment:
Why do we need this change?
It seems already logging the summary with info.

Reply:
@feeblefakie san,

Scalardl kelpie-test overrides this method.

https://github.com/scalar-labs/kelpie-test/blob/master/scalardl-test/src/main/java/scalardl/transfer/TransferReporter.java#L13

Reply:
Isn't it a bug of TransferReporter?
It doesn't make sense to me to update the base.

Reply:
Making TransferReporter not override `execute` causes any issues?